pkg/fixtures/writer_plain_mysql.go

```diff
@@ -0,0 +1,81 @@
+package fixtures
+
+import (
+	"fmt"
+	"github.com/Masterminds/squirrel"
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/db"
+	"github.com/applike/gosoline/pkg/mon"
+	"reflect"
+)
+
+type PlainMySqlFixtureValue []interface{}
+
+type PlainMySqlFixture struct {
+	TableName string
+	Columns   []string
```

Comment:
should we add these as metadata to the factory instead too? i assume the columns are always the same for each fixture of a table?

Reply:
Done.